packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const PROJECT_NAME = getPaths().base.match(/[^/|\\]+$/)[0]
+
+const RENDER_YAML = (database) => {
+  return `services:
+- type: web
+  name: ${PROJECT_NAME}-web
+  env: static
+  buildCommand: yarn rw deploy render web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: NODE_VERSION
+    value: 14
+  routes:
+  - type: rewrite
+    source: /.redwood/functions/*
+# replace destination api url after first deploy to Render
+    destination: replace_me_with_api_url/*
```

Comment:
Just a thing I notice when I use the previews feature with this config. It's rewrite in the preview web build with the prod API build URL. I know the preview feature is not enable in this config, but is one of the main reason I like Render. Maybe we can find something to fix that by default.

Reply:
Hey @simoncrypta, thanks for the feedback! Unfortunately, we don't have a fix for this right now but we are working on improving our IaC/Yaml to do this automatically in the future.

I'm glad that you enjoy our preview environments.